src/styles/components/MasterHeader.scss

```diff
@@ -53,6 +54,16 @@ nav.navbar{
   }
   .btn-link{
     height: 40px;
+    padding-left: 0;
+    padding-right: 0;
+    width: 100%;
+    text-align: left;
+
+    .caret {
+      position: absolute;
+      right: 12px;
+      top: 19px;
```

Comment:
cool, sounds good. can we use css flex rules?

On Wed, Apr 6, 2016 at 7:15 PM Mohamed El Mahallawy <
notifications@github.com> wrote:

> In src/styles/components/MasterHeader.scss
> <https://github.com/quran/quran.com-frontend/pull/258#discussion_r58744962>
> :
>
> > @@ -53,6 +54,16 @@ nav.navbar{
> >    }
> >    .btn-link{
> >      height: 40px;
> > +    padding-left: 0;
> > +    padding-right: 0;
> > +    width: 100%;
> > +    text-align: left;
> > +
> > +    .caret {
> > +      position: absolute;
> > +      right: 12px;
> > +      top: 19px;
>
> im not entirely sure how i feel about position absolute and putting fixed
> pixels for it. I made this huge mistake in the previous version and only
> pained me alot. Especially when looking at mobile, etc. I'd hazard away
> from this please.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/quran/quran.com-frontend/pull/258/files/e00db328e72a70a6f672982312e611272f2e6d0a#r58744962>
>